rwlib/CMakeLists.txt

```diff
@@ -41,10 +41,9 @@ SET(RWLIB_SOURCES
 	"source/job/WorkContext.cpp"
 	)
 
-add_library(rwlib
+set_source_files_properties(source/gl/gl_core_3_3.c PROPERTIES COMPILE_FLAGS -w)
```

Comment:
glLoadGen seems unmaintained - its last commit was on 2015-11-12.

On macOS, there is also this warning:
`warning: 'NSAddImage' is deprecated: first deprecated in OS X 10.5
      [-Wdeprecated-declarations]`
<hr>
Although compiling it as C++ source is not technically correct, it works - and does not need an intermediate header.
Compiling it as C source would just introduce some more CMake setup for one file. 

As I'm not sure how to solve this properly, I can just drop f1fba2a71d544ffe0ce9036b23a413f20a3db84d for now and open an issue.
I don't think this is a high-priority issue.